src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1153,9 +1153,6 @@ namespace SurfaceGeometry {
         int MultSurfNum;
         std::string MultString;
         auto &WarningDisplayed = state.dataSurfaceGeometry->WarningDisplayed;
```

Comment:
`state.dataSurfaceGeometry->WarningDisplayed` is a bool. Should be value here instead of ref?

Reply:
Didn't come up in my search. I will fix it.